src/main/java/wooteco/subway/section/Sections.java

```diff
@@ -0,0 +1,135 @@
+package wooteco.subway.section;
+
+import wooteco.subway.section.exception.SectionCantDeleteException;
+import wooteco.subway.section.exception.SectionInclusionException;
+import wooteco.subway.section.exception.SectionNotFoundException;
+import wooteco.subway.station.Station;
+import wooteco.subway.station.exception.StationNotFoundException;
+
+import java.util.*;
+
+public class Sections {
+    private final List<Section> sections;
+
+    public Sections(final List<Section> sections) {
+        this.sections = new ArrayList<>(sections);
+    }
+
+    public boolean canAttachesAfterEndStation(SectionDto sectionDto) {
```

Comment:
```suggestion
    public boolean canAttachAfterEndStation(SectionDto sectionDto) {
```

1. 조동사 다음엔 동사 원형~ 😎
2. 도메인이 DTO를 아는 구조가 된 것 같은데 어떻게 생각하시나요?

Reply:
해당 dto가 클라이언트한테 돌려주는 view의 성질을 가지고 있는 dto가 아니고,  ``SectionDto``의 내부 값이 추가되거나 삭제된다고 하더라도 ``upStationId``와 ``downStationId``는 계속해서 가지고 있을 것이라고 생각했기 때문에 크게 상관없다고 생각했습니다!!  
하지만, 이런 이유가 있다고 하더라도 결합도가 생긴다는 점이 굉장히 걸리긴 합니다!! 그래서 수정했습니다!!